app/models/spree/shipment.rb

```diff
@@ -331,6 +331,15 @@ def after_ship
       fee_adjustment.finalize!
       send_shipped_email
       touch :shipped_at
+      update_order_shipment_state
+    end
+
+    def update_order_shipment_state
+      new_state = order.updater.update_shipment_state
+      order.update_columns(
+        shipment_state: new_state,
+        updated_at: Time.now,
```

Comment:
I think it's safe to also use `Time.zone.now`. Rails seems to transparently save it in UTC regardless of the app's timezone therefore, it's a thing less to worry about; always use `Time.zone.now`.

Reply:
Ah, I pulled it straight from Spree! Yeah, we want `Time.zone.now` :100: